packages/thirdweb/src/utils/arweave.ts

```diff
@@ -0,0 +1,32 @@
+const DEFAULT_GATEWAY = "https://arweave.net/{fileId}";
```

Comment:
this gateway is run by the Arweave team, it's like "ipfs.io/ipfs" (but for Arweave). I'm pretty sure all public gateway will be overloaded eventually, but since we don't have that infra, I think this public gateway would be somewhat sufficient 🤔